pallets/elections/src/lib.rs

```diff
@@ -2,11 +2,31 @@
 //! Currently, it's PoA where the validators are set by the root account. In the future, a new
 //! version for DPoS elections will replace the current one.
 //!
-//! ### Terminology
+//! # Terminology
 //! For definition of session, era, staking see pallet_session and pallet_staking.
-//! - Committee: Set of nodes that produce and finalize blocks in the session.
-//! - Validator: Node that can become a member of committee (or already is) via rotation.
-//! - ReservedValidators: Validators that are chosen to be in committee every single session.
+//! - committee ([`EraValidators`]): Set of nodes that produce and finalize blocks in the session.
+//! - validator: Node that can become a member of committee (or already is) via rotation.
+//! - `EraValidators::reserved`: immutable validators, ie they cannot be removed from that list.
+//! - `EraValidators::non_reserved`: validators that can be kicked out from that list
+//!
+//! # Kick out logic
+//! In case of insufficient validator's uptime, we need to make such validators are removed from
```

Comment:
"we need to make such validators are removed " -> "we need to remove such validators"

Reply:
Ok let me fix this

Reply:
done